File: python/piramide.py
from figuras3d import Figuras3D

class Piramide(Figuras3D):
    def __init__(self, base, altura):
        self.base = base
        self.altura = altura
        
    def calcularVolumen(self):
        areaBase = self.base**2
        volumen = (areaBase * self.altura)/3
        return volumen

File: Java/Piramide.java
public class Piramide extends Figuras3d {
    protected double areaDeBase;
    protected double altura;

    public Piramide(String nombre,double areaDeBase, double altura ){
        super(nombre);
        this.areaDeBase=areaDeBase;
        this.altura=altura;

    }

    @Override
    public double calcularVolumen(){
        //(base*altura)/3
        return (areaDeBase*altura)/3;
        
    }
}
